tests/infrastructure/instance_types/test_cluster_max_hotplug_ratio.py

```
@@ -0,0 +1,168 @@
+from typing import Any, ClassVar, Dict
```

Ruff (0.14.14)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)